text/70_performance_testing.md

```diff
@@ -2,7 +2,7 @@
 
 What, exactly, is performance testing?  Before we can define "performance testing", we first need to define "performance".  I assume that at this point in the book, everybody understands what "testing" means.
 
-Most technically savvy people will have a gut feeling for what performance means - systems that are "fast", that "don't take up _muh memory_", etc.  The more you try to define it, though, the more slippery the definition becomes.  More fast, compared to what?  What if a system uses lots of RAM, and a competing system uses less RAM but more hard drive space?  If System A returns an answer in three seconds, and System B returns an answer in five seconds, which is more performant?  An easy question to answer, unless the questions that they are answering are themselves different.  Performance is one of those concepts that is hard to pin down, because it will mean different things for different systems.
+Most technically savvy people will have a gut feeling for what performance means - systems that are "fast", that "don't take up _much memory_", etc.  The more you try to define it, though, the more slippery the definition becomes.  Faster, compared to what?  What if a system uses lots of RAM, and a competing system uses less RAM but more hard drive space?  If System A returns an answer in three seconds, and System B returns an answer in five seconds, which is more performant?  An easy question to answer, unless the questions that they are answering are themselves different.  Performance is one of those concepts that is hard to pin down, because it will mean different things for different systems.
```

Comment:
When I did my first pass on typos in #3, I thought this was just playful dialect on @laboon's part.  On closer inspection, though, I agree this change is clearer.  #20 removes the italics I added here.